### PR TITLE
chore(release): publish

### PR DESCRIPTION
**Pull request by marcelltoth**
- @stoplight/elements-demo@7.0.1
 - @stoplight/elements-core@7.1.7
 - @stoplight/elements-dev-portal@1.0.7
 - @stoplight/elements@7.0.5

## Changes

fix: scroll on toc click (#1597) 
 fix: relative links (#1598)

### CODE REVIEW

File: packages/elements-dev-portal/package.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
   },
   "dependencies": {
     "@fortawesome/free-solid-svg-icons": "^5.10.2",
-    "@stoplight/elements-core": "~7.1.6",
+    "@stoplight/elements-core": "^7.1.7",
```
Comment (mpodlasin) [inline packages/elements-dev-portal/package.json:50]: Nope :P

File: packages/elements/package.json [hunk 2]
```diff
@@ -48,7 +48,7 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.31",
     "@fortawesome/free-solid-svg-icons": "^5.14.0",
     "@fortawesome/react-fontawesome": "^0.1.11",
-    "@stoplight/elements-core": "~7.1.6",
+    "@stoplight/elements-core": "^7.1.7",
```
Comment (mpodlasin) [inline packages/elements/package.json:51]: Nope :P

**Comment by mmiask** [comment]
Does this release `elements-utils`?

File: demo/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stoplight/elements-demo",
-  "version": "7.0.0",
```
Comment (mmiask) [inline demo/package.json:3]: Do we care though? This is still private

File: packages/elements-dev-portal/package.json (hunk 1 above)
Comment (marcelltoth) [reply packages/elements-dev-portal/package.json:50]: Fuck lerna. I'm really looking for a nice monorepo tool 😅 Or maybe just moving back to linked versioning 😄

File: demo/package.json (hunk 3 above)
Comment (marcelltoth) [reply demo/package.json:3]: No, I can undo it.

**Comment by marcelltoth** [comment]
> Does this release `elements-utils`?

No, should it?

**Comment by mmiask** [comment]
> > Does this release `elements-utils`?
> 
> No, should it?

No, I was just confused by specifying its version in the PR description. All clear now 👍

**Review by mmiask — APPROVED** [review]

**Pull request merged**